viewer/src/views/threejs/cad/consumeSectorSimple.ts

```diff
@@ -61,7 +56,7 @@ export function consumeSectorSimple(
 
   const obj = new THREE.Mesh(geometry, materials.simple);
 
-  obj.name = `Quads ${sectorId}`;
```

Comment:
Replace with ${metadata.id} rather than removing. It's very useful for debugging...

Reply:
It has been added outside of the loop instead